core/src/main/java/org/apache/calcite/sql2rel/RelDecorrelator.java

```diff
@@ -388,7 +389,16 @@ public Frame decorrelateRel(RelNode rel) {
   }
 
   /**
-   * Rewrite Sort.
+   * Rewrite {@link LogicalSort}.
+   *
+   * @param rel Sort to be rewritten
+   */
+  public Frame decorrelateRel(LogicalSort rel) {
```

Comment:
if this method didn't exist before, there's no need to add it for binary compatibility.

Reply:
Done.